Cargo.lock

```diff
@@ -21142,6 +21151,20 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
```

Comment:
In the past we've kept the version of these crates which we're directly using in sync with the version used by wasmtime, but we bump them in separate follow-up PRs because they can require messy changes (as new Wasm features are added) and we don't want them to block the regular wasmtime upgrades.

So how about doing these separately?